pnr/openroad/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('openroad-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: openroad
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
+    git_rev: master
+    patches:
+      - odb-drt-disable-tests.patch
```

Comment:
What are these?

Reply:
Sorry for the confusion, I also plan to submit those patches upstream (after reworking them a little bit, for example: this particular one would need to be put behind a additional cmake flag).

Reply:
Ack, will submit a PR upstream once I rework the patch.